pages/data-cleanup.tsx

```diff
@@ -378,7 +382,7 @@ export default function DataCleanupPage(props) {
       ...formData,
       contactRecord: {
         ...formData.contactRecord,
-        [target.name]: target.value.trim(),
```

Comment:
I removed the `.trim()` here because it was preventing the user from creating trailing spaces. Example: I was typing in Referred by field and wanted to type "My Friend" but I couldn't make the space after "My" to then type "Friend". I'm honestly not worried about whitespace for these right now otherwise.